comp/logs/agent/config/config.go

```diff
@@ -243,12 +250,16 @@ func BuildHTTPEndpointsWithConfig(coreConfig pkgConfig.Reader, logsConfig *LogsC
 
 		main.Host = host
 		main.Port = port
-		main.UseSSL = useSSL
+		*main.UseSSL = useSSL
 	}
 
 	additionals := logsConfig.getAdditionalEndpoints()
 	for i := 0; i < len(additionals); i++ {
-		additionals[i].UseSSL = main.UseSSL
+		if additionals[i].UseSSL == nil {
+			additionals[i].UseSSL = main.UseSSL
+		} else {
+			*additionals[i].UseSSL = !*additionals[i].UseSSL
```

Comment:
I don't think we want to flip this - I think this can be removed when the additional endpoint setting is renamed (from `logs_no_ssl` -> `use_ssl`)?